exts/betterTooltip/betterTooltip.js

```diff
@@ -82,6 +203,20 @@
                 return _seriesLabelFormatString;
             };
 
+            /**
+             * Gets or sets the format function for the label of the series.
+             *
+             * The default format function uses the seriesLabelFormatString property for creating the label.
+             *
+             * Custom formaters receive the full tooltip model object, the series being formated and the
```

Comment:
Custom formatters receive the full tooltip model object, the series being formatted and the